boa/contracts/vyper/vyper_contract.py

```diff
@@ -163,6 +173,14 @@ def abi(self):
     def _constants(self):
         return ConstantsModel(self.compiler_data)
 
+    def verify(self, verifier=None, license_type: str = None) -> None:
```

Comment:
kind of duplicated with `boa.verify()` right?

Reply:
https://github.com/vyperlang/titanoboa/pull/308/commits/a0a8a67e6b926ca4e2826df65d8d173aa6293047